src/e-cherry-pick.js

```diff
@@ -157,90 +167,67 @@ program
       const patchPath = `patches/${patchDirName}`;
       const targetBranches = [targetBranch, ...additionalBranches];
 
+      d(`Cloning electron/electron to ${tmp}`);
+      cp.execSync('git clone https://github.com/electron/electron', {
```

Comment:
Can we add a log for cloning? And enable stdio, some folks need stdio for ssh passphrases